aries-site/src/pages/design-tokens/overview.mdx

```diff
@@ -0,0 +1,100 @@
+- Build a shared language across design and development
+
+## Types of design tokens
+
+Tokens are grouped into 3 levels: [primitive](#primitive-tokens), [semantic](#semantic-tokens), and [component](#component-tokens). Each level exists to convey specificity, intended usage, and enhance token consumption and delivery. The name and intended use of a token becomes more specific as you move from primitive to semantic to component levels.
+
+<Notification
+  message={`When consuming design tokens, begin with the most specific token level available. Look first to see if a component token exists. If not available, use semantic tokens as a fallback.`}
+  status="info"
+  width={{ max: 'large' }}
+  margin={{ bottom: 'medium' }}
+/>
+
+### Primitive tokens
+
+Primitive tokens define the most foundational level of the token system, establishing fundamental attributes like color and dimension.
+
+The value of a primitive token will always be a CSS value such as a hex code, pixel, or rem value. Typically, primitive token names describe the token value and not where it should be applied.
+
+Primitive design tokens should not be used directly. They exist to provide the foundational options and be referenced in semantic and component design tokens.
+
+<Example height={{ min: 'xsmall' }}>
+  <Box align="center" direction="row" gap="small">
+    <Box background="brand" pad="medium" round="small" flex={false} />
+    <Text size="large">hpe.base.color.green.600</Text>
+  </Box>
+</Example>
+
+### Semantic tokens
+
+Semantic tokens apply a specific meaning to a primitive token to establish how and when a style should be applied. Semantic tokens capture design decisions for color, spacing, elevation, typography, and more.
+
+The value of a semantic token can either be a reference to a primitive token or another semantic token.
+
+Semantic tokens can have different values in different contexts. For example, the value of `hpe.color.background.back` might differ for light versus dark mode. Similarly, the value of `hpe.spacing.large` might differ for wide versus narrow viewports. By building with semantic tokens, the values can flex based on the context.
+
+<Example height={{ min: 'xsmall' }}>
+  <Box align="center" direction="row" gap="small">
+    <Box background="brand" pad="medium" round="small" flex={false} />
+    <Text size="large">hpe.background.primary.default</Text>
+  </Box>
+</Example>
+
+### Component tokens
+
+Component tokens capture specific design decisions for an individual component (such as button) or family of related components (such as inputs).
+
+The value of a component token can either be a reference to a primitive token, semantic token, or another component token.
+
+<Example height={{ min: 'xsmall' }}>
+  <Box align="center" direction="row" gap="small">
+    <Box background="brand" pad="medium" round="small" flex={false} />
+    <Text size="large">hpe.button.primary.enabled.background</Text>
+  </Box>
+</Example>
```

Comment:
Would having a CTA [similar to this](https://design-system.hpe.design/templates/page-layouts#layouts-for-page-content) recommending "How to read design tokens" be useful here?

Reply:
Done.